cfgov/jinja2/v1/_includes/organisms/ask-search.html

```diff
@@ -57,16 +55,12 @@
                                 input-contains-label
                                 {% if autocomplete %}m-autocomplete{% endif %}"
                          data-language="{{ language }}">
-                        {% set label_element = 'span' if show_label else 'label' %}
-                        {% set label_for = '' if show_label else 'for="o-search-bar_query"' %}
-                        <{{ label_element }} {{ label_for }}
+                        <label for="o-search-bar_query"
                                class="input-contains-label_before
                                       input-contains-label_before__search">
                             {{ svg_icon('search') }}
-                            {% if not show_label %}
                             <span class="u-visually-hidden">{{ _('Search') }}</span>
```

Comment:
Since we're getting rid of the old label, should we change this label to "Search for your question"? I'm not sure why I didn't use that as the label back in #6653, honestly. I guess we'd have to make sure the Spanish translation came through on Spanish pages, but I don't see why it wouldn't.

Reply:
Fixed in e3e9bd9